### PR TITLE
Internal docs

### PR DESCRIPTION
**Pull request by oblm**
**Contributions**: We are not currently taking public contributions - see our [contributions](CONTRIBUTING.md) policy. However, we are accepting issues and we do want your [feedback](../README.md#give-us-feedback).

-------

#### Description
* Added the `internal-docs` directory.
* Added the `release-process` doc.
* Added the `CHANGELOG`

#### Release note
Feature: Added a Changelog.

#### Tests
* Rendered all markdown,
* Executed the release process myself for 0.3.0.

#### Documentation
This is documentation.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,168 @@
+# Changelog
+All notable changes to the SpatialOS Game Development Kit for Unreal will be documented in this file.
+
+The format of this Changelog is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased] - 2019-xx-xx
```
Comment (joshuahuburn) [inline CHANGELOG.md]: Should we preemptively add the `Features`, `Bug fixes` and `Known Issues` sections so we have a place to put the new ones.

Comment (joshuahuburn) [reply CHANGELOG.md]: Ignore me

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+
+### Breaking Changes:
+- The GDK now uses SpatialOS 13.6.2. You **must** change the version specified in your projects `spatialos.json` file.
```
Comment (Vatyx) [inline CHANGELOG.md]: > You **must** change

Isn't that version number unused?

Comment (oblm) [reply CHANGELOG.md]: Good spot, I've removed this call to action.

**Pull request merged**